src/PrestaShopBundle/Form/Admin/Type/EventListener/ReductionListener.php

```diff
@@ -53,7 +53,8 @@ public static function getSubscribedEvents(): array
     public function adaptReductionField(FormEvent $event): void
     {
         $data = $event->getData();
-        if ($data['type'] === Reduction::TYPE_PERCENTAGE) {
+
+        if (isset($data['type']) && $data['type'] === Reduction::TYPE_PERCENTAGE) {
```

Comment:
without parenthesis I'm not sure of the priority beetwen && and === ... are you ?

Reply:
yep im sure, its just 2 simple comparisons, don't think its worth wrapping in parentheses.
1. isset($data['type']),
2. $data['type'] === Reduction::TYPE_PERCENTAGE

with parentheses it would look like this (which doesnt make it anymore readable IMO)
```
if (isset($data['type']) && ($data['type'] === Reduction::TYPE_PERCENTAGE))
```

Reply:
ok go !